common.gypi

```diff
@@ -211,6 +214,9 @@
         # and their sheer number drowns out other, more legitimate warnings.
         'DisableSpecificWarnings': ['4267'],
         'WarnAsError': 'false',
+        'AdditionalOptions': [ 
+          '/std:c++14'
```

Comment:
Unnecessary, both VS2015 and VS2015 default to `/std:c++14`.

Reply:
I'd rather have this (1) for being explicit (2) for future proofing, in case the default changes to C++17 or higher